### PR TITLE
Expand on `query_params` for ObjectVar in the documentation

### PR DESCRIPTION
**Pull request by Kircheneer**
# Closes: DNE
# What's Changed

Expand on `query_params` for `ObjectVar` in the documentation, clarify that its syntax is composed of REST API parameters rather than ORM ones.

### CODE REVIEW

File: nautobot/docs/additional-features/jobs.md [hunk 1]
```diff
@@ -304,16 +304,20 @@ device = ObjectVar(
 )
 ```
 
-Multiple values can be specified by assigning a list to the dictionary key. It is also possible to reference the value of other fields in the form by prepending a dollar sign (`$`) to the variable's name.
+Multiple values can be specified by assigning a list to the dictionary key. It is also possible to reference the value of other fields in the form by prepending a dollar sign (`$`) to the variable's name. You can even traverse the model relationships this way (note the single underscore in the example - `tenant_region_id`).
```
Comment (glennmatthews) [inline nautobot/docs/additional-features/jobs.md:307]: Not exactly - `tenant_group_id` is an [explicitly defined parameter on the FilterSet](https://github.com/nautobot/nautobot/blob/develop/nautobot/tenancy/filters.py#L71) (in this case, the `TenancyFilterSet` mixin that `SiteFilterSet` inherits from, so it works; `tenant_region_id` is not a defined FilterSet parameter so it won't work.

Comment (Kircheneer) [reply nautobot/docs/additional-features/jobs.md]: Alright I reworded this a little and changed the `tenant_region_id` to be what I mean to: `tenant_group_id`.

**Review by glennmatthews — APPROVED** [review]
Thanks!

**Comment by jathanism** [comment]
Oops, this was supposed to go into next. Womp womp.

**Comment by jathanism** [comment]
Merged into next.

**Pull request merged**